programs/clearing_house/src/math/amm.rs

```diff
@@ -917,14 +940,22 @@ pub fn calculate_budgeted_k_scale(
     market: &mut Market,
     budget: i128,
     _mark_price: u128, // todo
+    increase_max: i128,
 ) -> ClearingHouseResult<(u128, u128)> {
+    let curve_update_intensity = market.amm.curve_update_intensity as i128;
```

Comment:
would be nice to have this logic inside the test coverage. although unlikely there can still be bugs in this code that get missed because it isn't in `_calculate_budgeted_k_scale` test coverage

Reply:
added